code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -215,15 +215,18 @@
 	SSblackbox.record_feedback("tally", "cyborg_modules", 1, R.module)
 
 /**
- * check_menu: Checks if we are allowed to interact with a radial menu
+ * Checks if we are allowed to interact with a radial menu
  *
  * Arguments:
- * * user The mob interacting with a menu
+ * * user The cyborg mob interacting with the menu
+ * * old_module The old cyborg's module
  */
-/obj/item/robot_module/proc/check_menu(mob/user)
+/obj/item/robot_module/proc/check_menu(mob/living/silicon/robot/user, obj/item/robot_module/old_module)
 	if(!istype(user))
 		return FALSE
-	if(user.incapacitated() || !user.Adjacent(src))
+	if(user.incapacitated())
```

Comment:
I didn't notice before merging but was the Adjacent check unnecessary here ?

Reply:
It was - We are already handling that in the radial menu with parameter require_near = TRUE. We were basically checking twice.